templates/rust-starter/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "methods-guest"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+{% assign feature_std = ", features = ['std']" -%}
+
+[build-dependencies]
+{% if risc0_crate_branch -%}
+risc0-build = { git = "https://github.com/risc0/risc0.git", branch = "{{ risc0_crate_branch }}" }
+{% else %}
+risc0-build = "{{ risc0_version }}"
+{% endif %}
+[dependencies]
+{% unless risc0_std %}
+# If you want to try (experimental) std support, add `features = [ "std" ]` to risc0-zkvm
```

Comment:
I think its still helpful for users who don't use `--std` because the default in `new` is no_std but many folks would probably end up adding it later on. At least thats my thought.